site/src/components/Loader/Loader.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { Loader } from "./Loader";
+
+const meta: Meta<typeof Loader> = {
+  title: "components/Loader",
+  component: Loader,
+};
+
+export default meta;
+type Story = StoryObj<typeof Loader>;
+
+export const Example: Story = {};
+
+export const Fullscreen: Story = {
+  args: {
+    fullscreen: true,
+  },
+};
```

Comment:
Am I missing something? These stories are showing up blank for me - probably because it's an animation? Perhaps we are just trying to test the fullscreen nature of the container. My worry is that doing this through Chromatic makes it look like the story isn't rendering. 

We could try fiddling with `pauseAnimationAtEnd` or introducing a pause as documented [here](https://www.chromatic.com/docs/animations/).

WDYT about renaming 'Example' to something a little more descriptive or adding a description param?

Reply:
Oh, supposedly Chromatic will pause SVG animations automatically. Let's see what havoc this wreaks.